.claude/rules/release.md

```
@@ -0,0 +1,253 @@
+pnpm run typecheck && pnpm run biome:check && pnpm run test
+```
+
+**Requirements for the Ship Validation phase:**
+- All TypeScript type checks pass (zero errors)
+- All Biome lint and formatting checks pass (zero errors)
+- All unit tests pass
+- If any check fails, fix before proceeding to PR creation
+
+## Branch Strategy
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)